Working Mode.md

```diff
@@ -147,6 +147,8 @@ Implementation disagreement should not result in implementation-defined behavior
 
 Most Workstreams don't run any regular meetings. However, as circumstances arise, meetings can be helpful to resolve difficult issues, or to liaise with other groups.
 
+There is a regular triage meeting held approximately every other week to unblock issues across all WHATWG Workstreams.  Topics may come from any WHATWG repository; you can find the information on the next meeting by looking for the [Upcoming WHATNOT meeting](https://github.com/whatwg/html/issues?q=is%3Aissue+is%3Aopen+label%3Aagenda%2B+Upcoming) in the HTML repository.  Topics are added to the agenda for the next meeting by adding the "agenda+" label in any WHATWG repository.  The timing of this meeting changes to distribute across time zones.
```

Comment:
Nits:

* Single space after periods.
* Either HTML standard repository or `html` repository, prolly the former.
* "in any WHATWG standard repository" (we don't add labels to all repositories)

Reply:
All fixed.  (Sorry, I have an involuntary two-space-after-period tic, of course I intended to match style.  :) )